frontend/src/Troop/TroopInformation.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { View, Image, Text } from 'react-native';
+import commonStyles from '../common/styles';
+import troopAvatar from '../../assets/troop/troop-avatar.jpg';
+
+function TroopInformation() {
+  return (
+    <View style={{ marginRight: 24 }}>
+      <Image style={{ width: 100, height: 100, borderRadius: 10 }} source={troopAvatar} />
```

Comment:
marginRight is OK in inline, but this is too long to put here, please separate the Image styling to a stylesheet as you did in the TroopLevel.jsx

Reply:
exactly, so I use another way to solve this problem.  you can checkout and run it.